inc/remove-admin-menu-items.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Remove admin menu pages.
+ *
+ * @package MEOM Dodo
+ */
+
+namespace MEOM\dodo;
+
+/**
+ * Remove admin menu items.
+ */
+function remove_admin_menu_items() {
```

Comment:
I would like to add a filter where you can decide when these items are removed from the menu.
Could we show these by default if we are not in production? My dream is that the users would have setting "show all menu items" or something, developers could enable that and save few seconds while doing upkeep tasks :)

Reply:
Yep, that's why I asked is Seravo using natine ENV variable :)

> My dream is that the users would have setting "show all menu items" or something, developers could enable that and save few seconds while doing upkeep tasks :)

Do you mean clients would have that setting also?

Reply:
Would it be WP_ENV? At least that one is used in a few occasions. You probably could also quickly test it by adding a value for it in .env and docker-compose.yml files and checking the container for what happened.

Reply:
Ah, yeh there is `WP_ENV`. I'll test that and try to add issue in Seravo stack that how it relates to native WP ENV-variable.

Reply:
> Do you mean clients would have that setting also?
Yes. In some (rare) cases it is possible that one user from the client needs easy access for the plugins.

Reply:
OK. I'll start with the ENV check first that in development we can see all the menu items.

Reply:
Added in 46391aca5d9fbdb95d6caf3a0348c449ffbb1747. Show all admin menu items if not in production with filter.

Reply:
@giraffewoods has a good point. It would be very good to see all menu items if I'm doing upkeep instead of switching to staging and go to some thing that is hidden in menu in production and then switch back to production to get there.

Reply:
@jjfer That's indeed coming in PR #3 as user setting.

There is also filters for hard core users :)